src/CreateHandler/CreateHandler.php

```diff
@@ -132,19 +134,22 @@ private function getComposerAutoloaders(): array
             throw CreateHandlerException::invalidComposerJson($jsonException->getMessage());
         }
 
-        if (! isset($composer['autoload']['psr-4'])) {
+        if (! is_array($composer) || ! isset($composer['autoload']['psr-4'])) {
```

Comment:
Check is not needed here. Annotate `$composer` returned from `json_decode` instead imo.